routes/api/authRouter.js

```diff
@@ -108,4 +130,29 @@ router.get("/current", authMiddleware, async (req, res, next) => {
   }
 });
 
+router.patch("/avatars", authMiddleware, upload.single('avatar'), async (req, res, next) => {
+  try {
+    const userId = req.user._id;
+
+    if (!req.file) {
+      return res.status(400).json({ message: 'No file uploaded' });
+    }
+
+    const Jimp = require('jimp');
```

Comment:
Importuj rzeczy na górze pliku, tak jak teraz to wtedy jeśli potrzebujesz dynamicznie coś importować.